nextcord/types/role.py

```diff
@@ -27,3 +27,6 @@ class RoleTags(TypedDict, total=False):
     bot_id: Snowflake
     integration_id: Snowflake
     premium_subscriber: None
```

Comment:
Shouldn't these be `NotRequired[None]`?

Reply:
The class has total=False, so all are not required